build.yaml

```diff
@@ -14,7 +14,7 @@ cassandra:
   - 2.1
   - 2.2
   - 3.0
-  - 3.10
+  - 3.7
```

Comment:
Only reason I knew that was because I had also ran into this recently: https://github.com/datastax/java-driver/blob/3.1.x/build.yaml#L39-L44